src/utils.ts

```diff
@@ -14,3 +16,11 @@ export function assertRequired<T>(object: T, ...properties: (keyof T & string)[]
     }
   })
 }
+
+export async function logFirstUsage() {
```

Comment:
move this into it's own module `logger.ts`
introduce a constant for `nosto:web-components:logged`
and use the constant while querying localStorage.